src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,58 @@
+package lotto.domain;
+
+import java.util.List;
+
+public class Lotto {
+
+    public static final int PRICE = 1_000;
+
+    public static final int SIZE = 6;
+
+    public static final int MIN = 1;
+
+    public static final int MAX = 45;
+
+    public final List<Integer> numbers;
+
+    public Lotto(List<Integer> numbers) {
+        validateNumbers(numbers);
+        this.numbers = numbers;
+    }
+
+    private void validateNumbers(List<Integer> numbers) {
+        if (!isValidLottoSize(numbers)) {
+            throw new IllegalArgumentException("여섯자리 번호를 입력해 주세요");
+        }
+        if (hasDuplicate(numbers)) {
+            throw new IllegalArgumentException("로또 번호는 중복된 번호를 입력할 수 없습니다.");
+        }
+        if (containsInvalidNumber(numbers)) {
+            throw new IllegalArgumentException("로또 번호는 1에서 45사이의 숫자값만 입력할 수 있습니다.");
+        }
+    }
+
+    private boolean isValidLottoSize(List<Integer> numbers) {
+        return numbers.size() == Lotto.SIZE;
+    }
```

Comment:
다음과 같이 해줘도 좋을 것 같아요! 😄 
```suggestion
    private boolean isValidLottoSize(List<Integer> numbers) {
        return numbers.size() == SIZE;
    }
```